### DISCUSSION TITLE
NAV does not turn on

### DISCUSSION THREAD

**Discussion by escoces103** [flybywiresim/aircraft#6082] [General]
hi all

I have the 320nx stable version,  0.7.0, since last msfs2020 update NAV does not turn on, even after the irs alignment, so the auto pilot does not follow the flight plan path
does anybody have the same issue?

thanks

**Comment by Benjozork** [comment]
Did you pull for managed heading ?

**Reply by escoces103** [reply]
yes, but does nothing
